learnEcl/2700-function.md

```diff
@@ -2,3 +2,147 @@
 title: Function
 slug: function
 ---
+
+# FUNCTION
+
+A Function is a set of statements that take inputs, does some specific computation and produces an output or return a result. Result could be a value or a dataset.
```

Comment:
*Optionally* takes inputs (you can write functions that don't require arguments).

Also, functions can return actions (e.g. OUTPUT).